tree.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable import/prefer-default-export */
+
+class Node {
+    constructor(data) {
+      this.data = data;
+      this.left = null;
+      this.right = null;
+    }
+  }
+  export class Tree {

```

Comment:
use jsdoc to document classes and methods

Reply:
Made the changes.